api/neo4j/neo4jApi.py

```diff
@@ -61,6 +68,10 @@ def get_followers(user_ids):
             followers_dictionary.update({user:[follower]})
         else:
             followers_dictionary[user].append(follower)
+    for uid in user_ids:
+        if uid not in followers_dictionary:
```

Comment:
You assume here that users not in followers_dictionary are still unknown to us. That is not necessarily the case! It only means that they have no followers inside this list of uids... why not query the database to check if these guys have no followers yet (and are still "unknown")?